packages/base/src/3dview/mainview.tsx

```diff
@@ -1257,6 +1283,9 @@ export class MainView extends React.Component<IProps, IStates> {
             : 'unset'
         }}
       >
+        <button onClick={this.toggleWireframe}>
+            Toggle Wireframe
+          </button>
```

Comment:
Just temporarily added this button for testing. Further plan is to add this in toolbar and also have a key binding for the same.